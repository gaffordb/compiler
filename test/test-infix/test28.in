let x : int = 5 in
let y : int = 2 in
fst (x . y) + snd (x . y)
