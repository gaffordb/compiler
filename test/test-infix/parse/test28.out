(let x = 5 in
(let y = 2 in
(+ (fst (x . y)) (snd (x . y)))))
